relentless/potential/potential.py

```diff
@@ -40,29 +41,33 @@ class CoefficientMatrix(PairMatrix):
 
     Set coefficient matrix values::
 
-        m['A','A']['energy'] = 2.0
-        m['A,'B']['mass'] = 1.5
-        m['A','B']['energy'] = 3.0
+        m['A','A']['energy'] = 2.0               #access param directly
+        m['A','B'] = {'energy':0.0, 'mass':1.5}  #reset default in full
+        m['A','A'] = {'mass':0.5}                #reset default partially
+        for p in self.params:                    #iteratively acesss params
+            m['B','B'][p] = 0.1
 
     Evaluate (retrieve) pair parameters::
```

Comment:
Similarly, can you also show an example of what happens when you do `m.evaluate(('A','A'))` vs. accessing `m['A','A']`?